src/analytics/controllers/dashboard.py

```diff
@@ -215,7 +215,7 @@ def download_customised_data():
             return final_data
 
 
-@dashboard_bp.route('/api/v1/dashboard/customisedchart/random/chartone', methods=['GET'])
+@dashboard_bp.route('/api/v1/analytics/dashboard/customisedchart/random/chartone', methods=['GET'])
```

Comment:
@ja-odur , not sure weather this is still being consumed anywhere. may be it can go away